react/tic-tac-toe/src/redux/actions.js

```diff
@@ -0,0 +1,41 @@
+import TicTacToeService from "../services/TicTacToeService";
+import { createTypes, completeTypes, withPostFailure, withPostSuccess} from 'redux-recompose';
+
+
+export const actions = createTypes(completeTypes(['GET_MATCHES','LOG_IN']), '@TICTACTOE');
+
+const actionCreators = {
+  getMatches: () => {
+    return {
+      type: actions.GET_MATCHES,
+      service: TicTacToeService.getMatches,
+      target: 'matches',
+      payload: {},
+      injections: [
+        withPostFailure(() => {
+          window.alert("Error while loading the matches");
```

Comment:
Instead of alert you should show an error message in the corresponding view (where the matches should be)